modules/programs/zsh.nix

```
@@ -482,7 +484,7 @@ in
     }
 
     {
-      home.packages = with pkgs; [ zsh ]
+      home.packages = with pkgs; [ cfg.package ]
         ++ optional cfg.enableCompletion nix-zsh-completions
         ++ optional cfg.oh-my-zsh.enable cfg.oh-my-zsh.package;
```

Seems cleaner to get rid of the `with` because it's only being used by one of thing here.

```suggestion
      home.packages = [ cfg.package ]
        ++ optional cfg.enableCompletion pkgs.nix-zsh-completions
        ++ optional cfg.oh-my-zsh.enable cfg.oh-my-zsh.package;
```